### PR TITLE
Add info about CHANGELOG on the CONTRIBUTING

### PR DESCRIPTION
**Pull request by maurogeorge**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,8 +24,10 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Write tests for your feature, or regression tests highlighting a bug
 4. Write the feature itself, or fix your bug
-5. Commit your changes (`git commit -am 'Add some feature'`)
-6. Push to the branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
+5. Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
+with a brief description of your changes under the current develop version heading.
```
Comment (bf4) [inline CONTRIBUTING.md]: would love to note somehow that if there's a merge conflict with the CHANGELOG in a pr (which is frequent), the maintainers will ensure the pr is merged

Comment (bf4) [inline CONTRIBUTING.md]: `with a brief description of any breaking changes, fixes, features, or miscellaneous changes under the proper version section.`

Comment (bf4) [inline CONTRIBUTING.md]: btw, :+1: thanks for this

**Comment by beauby** [comment]
:+1: Let's get the wording right, and then merge it.

**Comment by maurogeorge** [comment]
Thanks for the review. Updated with the right wording.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,8 +24,11 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Write tests for your feature, or regression tests highlighting a bug
 4. Write the feature itself, or fix your bug
-5. Commit your changes (`git commit -am 'Add some feature'`)
-6. Push to the branch (`git push origin my-new-feature`)
-7. Create a new Pull Request
+5. Update [CHANGELOG.md](https://github.com/rails-api/active_model_serializers/blob/master/CHANGELOG.md)
```
Comment (beauby) [inline CONTRIBUTING.md]: The problem if you update the changelog at that point is that you don't know the PR reference yet.

Comment (joaomdmoura) [inline CONTRIBUTING.md]: :+1: maybe update change log and squash unnecessary commits should be done later in this process

Comment (beauby) [inline CONTRIBUTING.md]: Usually, the following happens:
 1. Create your branch and commit the feature/tests, push and issue PR
 2. Iterate on feedback given by the community (fix syntax, modify bits of code, add tests), pushing the new commits to the PR each time
 3. When it looks ready to merge, update the changelog + squash the commits

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,8 +24,11 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
...
+with a brief description of any breaking changes, fixes, features, or
+miscellaneous changes under the proper version section.
+6. Commit your changes (`git commit -am 'Add some feature'`)
+7. Push to the branch (`git push origin my-new-feature`)
+8. Create a new Pull Request
 
 Remember to squash your commits and rebase off `master`.
```
Comment (NullVoxPopuli) [inline CONTRIBUTING.md]: Maybe for super new contributors, we could add  how to do this off upstream/master

like, I rebase this way:
 - squash commits on my branch (to reduce the number of repeated conflicts if any as git is applying changes) via `git rebase -i HEAD~N` where N is the number of commits I've made on the branch (found by counting commits in `git log`
 - update master branch with upstream's code. `git checkout master; git fetch upstream; git reset upstream/master --hard`
 - rebase against master: `git rebase master`
 - push rebased code: `git push origin {branch name} -f`

Comment (NullVoxPopuli) [inline CONTRIBUTING.md]: oh, and also, at no point should master be *merged* in to the feature branch.  That's what messed me up early on.

**Comment by maurogeorge** [comment]
Thanks for the review.

I update the code, take a look when you have some time :smile:

**Comment by joaomdmoura** [comment]
Nice @maurogeorge merging it.

**Comment by maurogeorge** [comment]
Thanks for the merge :+1:

**Pull request merged**